documentation/src/main/asciidoc/stories/assembly_wildfly_modules.adoc

```diff
@@ -0,0 +1,21 @@
+[id='ispn_modules']
+:context: server
+= {brandname} Modules for {wildflybrandname}
+To use {brandname} inside applications deployed to {wildflybrandname}, you should install {brandname} modules that:
+
+* Let you deploy applications without packaging {brandname} JAR files within your WAR, EAR, and so on.
+* Use different slots to the internal {wildflybrandname} modules for {brandname}, which avoids conflicts.
```

Comment:
I think this can be more generic. Something like:

"Allows you to utilise a {brandname} version independent of that bundled with {wildflybrandname}."